docs/infrastructure.md

```diff
@@ -172,279 +170,213 @@ When you launch the graph backend for the first time,
-        since the existing permissions list will be overwritten:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1", "WRITE_REPO_my_db2","READ_REPO_my_db2"]}`
+!!! info
+    This command is also run as part of the [`graphdb_setup.sh`]() script for first-time GraphDB backend setup.
 
-        Similarly, to revoke `my_db1` access for `DBUSER`, so they only have access to `my_db2`, 
-        you would use the following permissions list:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db2","READ_REPO_my_db2"]}`
+To grant `DBUSER` read/write access to a second database `my_db2` (while keeping the existing access to `my_db1`), 
+you would rerun the above `curl` command with _all_ permissions (existing and new) specified since the existing permissions list will be overwritten.
 
-        You can verify a user's settings at any time with the following:
-        ```bash
-        curl -u "admin:NewAdminPassword" http://localhost:7200/rest/security/users/DBUSER
-        ```
+So, your new provided permissions list would look as follows:
+```json
+{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1", "WRITE_REPO_my_db2","READ_REPO_my_db2"]}
+```
 
-    !!! info "Managing user permissions using the GraphDB Workbench"
+Similarly, to revoke `my_db1` access so `DBUSER` only has access to `my_db2`, 
+you would use the following permissions list:
+```json
+{"grantedAuthorities": ["WRITE_REPO_my_db2","READ_REPO_my_db2"]}
+```
 
-        If you are managing multiple GraphDB databases, the web-based administration interface for a GraphDB instance, the Workbench, 
-        might be an easier way to manage user permissions than the REST API.
-        More information on using the GraphDB Workbench can be found [here](https://graphdb.ontotext.com/documentation/10.0/workbench-user-interface.html).
+!!! tip "Managing user permissions using the GraphDB Workbench"
 
-=== "Stardog"
+    If you are managing multiple GraphDB databases, the web-based administration interface for a GraphDB instance, the Workbench, 
+    might be an easier way to manage user permissions than the REST API.
+    More information on using the GraphDB Workbench can be found [here](https://graphdb.ontotext.com/documentation/10.0/workbench-user-interface.html).
 
-    ```bash
-    curl -X POST -i -u "admin:NewAdminPassword" http://localhost:5820/admin/databases \
-    --form 'root="{\"dbname\":\"test_data\"}"'
-    ```
+### Resetting your GraphDB instance
 
-    Now we need to give our new database user read and write permission for 
-    this database:
+If you previously started/completed the first-time graph setup for Neurobagel (i.e., you previously set up a Neurobagel node on your machine) but want to set up your graph database again _from scratch_, 
+the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any credentials or databases that were been previously created.
```

Comment:
```suggestion
If you have already set up a Neurobagel node on your machine but want to reset your graph database to start again _from scratch_, 
the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any credentials or databases that were been previously created.

Here are some reasons you might want to reset your GraphDB instance:
- your graphDB instance has stopped working after a configuration change (i.e. your neurobagel node API no longer responds or responds with an error)
- ...
```

Two main comments:
- simplify the text AND
- give some context to WHY I would like to do that. Just saying how I can do it only helps if I already know that I want to reset graphDB - most likely I have no clue that this is what I want